nengo/__init__.py

```diff
@@ -26,11 +26,13 @@
 from .learning_rules import PES, BCM, Oja, Voja
 from .params import Default
 from .probe import Probe
+from .processes import (BrownNoise, FilteredNoise, PresentInput, Process,
+                        WhiteNoise, WhiteSignal)
```

Comment:
Hmm... yeah, looking back at this, I'm leaning towards just keeping the `processes` module import and having people use the full name. The `dists` name is intentionally short so that it isn't too bad to do `nengo.dists.whatever`. But yeah, we should maybe be more consistent and import either none or all of a particular module, so I'll make sure we're doing that.